backend/src/services/MergeActionsService.ts

```diff
@@ -0,0 +1,34 @@
+import { LoggerBase } from '@crowd/logging'
+import { queryMergeActions } from '@crowd/data-access-layer/src/mergeActions/repo'
+import { IServiceOptions } from './IServiceOptions'
+import SequelizeRepository from '@/database/repositories/sequelizeRepository'
+
+export default class MergeActionsService extends LoggerBase {
+  options: IServiceOptions
+
+  constructor(options: IServiceOptions) {
+    super(options.log)
+    this.options = options
+  }
+
+  async query(args) {
+    const qx = SequelizeRepository.getQueryExecutor(this.options)
+    const results = await queryMergeActions(qx, args)
+
+    return results.map((result) => ({
+      primaryId: result.primaryId,
+      secondaryId: result.secondaryId,
+      state: result.state,
+      // derive operation type from step and if step is null, default to merge
+      operationType: result.step ? MergeActionsService.getOperationType(result.step) : 'merge',
+    }))
+  }
+
+  static getOperationType(step) {
+    if (step.startsWith('merge')) {
+      return 'merge'
+    }
+
+    return 'unmerge'
+  }
```

Comment:
NIT: This will also return `unmerge` for steps that don't start with `unmerge`. Maybe add another else if and throw an error if the step is not recognized?

Reply:
+1, good idea. I will update the PR.